typings.d.ts

```diff
@@ -245,7 +277,25 @@ declare module 'resin-components' {
 
 	class Text extends RenderableElementWithProps<DefaultProps, any> {}
 
-	interface TextareaProps extends StyledSystemProps, React.TextareaHTMLAttributes<HTMLTextAreaElement> {}
+	interface TextareaProps extends DefaultProps {
+		autoComplete?: string;
```

Comment:
This could even be `'on' | 'off'`

Reply:
The [spec defines way more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-autocomplete) than just those two...

Reply:
This is taken from the react typings, I'm happy to leave it as is due to the large variety of possible values

Reply:
> The spec defines way more than just those two...

That's for `<input>`, not `<textarea>`, which has only two: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea#attr-autocomplete

Reply:
That said, it doesn't make much of a difference, so if we want to leave this as `string` that's fbm too.